### DISCUSSION TITLE
Midtrans payment gateway

### DISCUSSION THREAD

**Discussion by pasarsegar** [inspireui/support#5518] [Request new Native Payment]
Native gateway for Midtrans would be nice. Thanks.
https://pub.dev/packages/midtrans_sdk

**Comment by byndrha** [comment]
supported already?

**Comment by pasarsegar** [comment]
Seems not yet. They are very slow.
  

  

  

  
  
  
  
>

**Comment by minhtruong315** [comment]
Hi @pasarsegar I'm from FluxStore team. I'm investigating the Midtrans native payment gateway to FluxStore app. But I don't have the Midtrans account. It requires the ID phone number, but I don't have it.
https://dashboard.midtrans.com/register
if you bought FluxStore app, please help me create new ticket in https://support.inspireui.com and send the Midtrans account to implement Midtrans native payment gateway.
if you haven't bought FluxStore app, you can send me the account via email: minhtruong315@gmail.com. 
Thanks so much!

**Comment by HanielQC** [comment]
Done on FluxStore app 3.8.0 
https://app.inspireui.com/woocommerce/changelog
